spec/javascript/components/applications/application_form/PaperApplicationForm.test.js

```diff
@@ -95,6 +96,43 @@ describe('PaperApplicationForm', () => {
         )
         expect(wrapper.text()).toContain('Eligibility Information')
       })
+
+      test('lenders should be filled out', async () => {
+        const wrapper = mount(
+          <PaperApplicationForm
+            listing={listing}
+            lendingInstitutions={lendingInstitutions}
+            application={applicationWithInvalidAnnualIncome}
+          />
+        )
+        wrapper.find('#lending_institution select').simulate('change', { target: { value: 1 } })
+        expect(wrapper.text()).toContain('Hilary Byrde')
+        wrapper.find('form').first().simulate('submit')
+        expect(wrapper.text()).toContain('Please select a lender.')
+        expect(wrapper.text()).toContain('The applicant cannot qualify for the listing unless this is true.')
+        wrapper.find('#lending_agent select').simulate('change', { target: { value: 1 } })
+        wrapper.find('form').first().simulate('submit')
+        expect(wrapper.text()).not.toContain('Please select a lender.')
+      })
+
+      describe('with lending agent field filled', () => {
```

Comment:
same here, I think it could be a bit clearer. Maybe something like "with lending agent id provided" -> "lending institution and lender dropdowns should be filled out"